js/jquery.plugincreator.es6

```diff
@@ -0,0 +1,290 @@
+/*!
+ * jQuery Plugin Creator 0.3.0
+ * https://github.com/OOPMan/jquery-plugin-creator
+ *
+ * Copyright 2014 Adam Jorgensen
+ * Released under the MIT license.
+ * http://github.com/OOPMan/jquery-plugin-creator/LICENSE
+ *
+ */
+/**
+ * TODO: Add checks to prevent over-writing plugins
+ * TODO: Add ability to not specify plugin name and have a randomly generated name assigned
+ */
+"use strict";
```

Comment:
@OOPMan http://stackoverflow.com/questions/31685262/not-recommended-to-write-out-use-strict-with-es6 correct